scenario:
  name: 'Scenario Camille - Homing'

  overview:  "0. Starting from [-4.0, -10.0, 0.0] drift to [-4.0, 8.0, 0.0].\n
              1. Do ARM1 and CM+ to reach the same orbit.\n
              2. Do ARM2 to go 2km behind the target on the same orbit.\n"

  keep_out_zone: [0.2, 0.2, 0.2]

  approach_ellipsoid: [0.5, 1.5, 0.5]

  prop_type: 2-body

  checkpoints:
    S0:
      id: 0
      state:
        lvlh:
          R: [-4.0, 8.0, 0.0]
          V: [0.0, 0.0, 0.0]
      error_ellipsoid: [0.4, 1.5, 0.4]
      manoeuvre: drift

    S1:
      id: 1
      state:
        lvlh:
          R: [0.0, 18.0, 0.0]
          V: [0.0, 0.0, 0.0]
      error_ellipsoid: [0.2, 1.0, 0.2]
      t_min: 2800
      t_max: 3600

    S2:
      id: 2
      state:
        lvlh:
          R: [0.0, -2.0, 0.0]
          V: [0.0, 0.0, 0.0]
      error_ellipsoid: [0.1, 0.3, 0.1]
      t_min: 29000
      t_max: 32000
